src/config/Config.zig

```diff
@@ -5752,13 +5732,11 @@ pub const GtkSingleInstance = enum {
 pub const GtkTabsLocation = enum {
     top,
     bottom,
-    left,
-    right,
```

Comment:
Again, thinking about users, maybe we should keep these around and default them to top?

Reply:
As above, I'd rather be noisy about it so that people know why their tabs aren't showing up where they expect them to.